com.dynamo.cr/com.dynamo.cr.bob.test/src/com/dynamo/bob/pipeline/ShaderProgramBuilderTest.java

```diff
@@ -200,11 +200,11 @@ public void testGlslDirectives() throws Exception {
                     "#extension GL_OES_standard_derivatives : enable\n" +
                     "precision mediump float;\n" +
                     "\n" +
-                    "out vec4 _DMENGINE_GENERATED_gl_FragColor;\n" +
+                    "out vec4 _DMENGINE_GENERATED_gl_FragColor_0;\n" +
```

Comment:
The GLSL transformation now supports multiple outputs, so the default will always be postfixed with _0.